Tests/LibWeb/test-web/main.cpp

```diff
@@ -644,6 +644,7 @@ static ErrorOr<int> run_tests(Core::AnonymousBuffer const& theme, Web::DevicePix
     for (size_t i = 0; i < concurrency; ++i) {
         auto view = TestWebView::create(theme, window_size);
         view->on_load_finish = [&](auto const&) { ++loaded_web_views; };
+        view->reset_zoom();
```

Comment:
Let's add a FIXME here. This is fine for now, but I think we want to more generally have a way to ensure test-web runs with default settings. We don't want to special-case every setting that affects tests here.